README.md

```diff
@@ -19,6 +19,15 @@ By
 [![WordPress Rating](https://img.shields.io/wordpress/plugin/r/timber-library.svg?style=flat-square)](https://wordpress.org/support/plugin/timber-library/reviews/)
 [![!Financial Contributors](https://opencollective.com/timber/tiers/badge.svg)](https://opencollective.com/timber/)
 
+> ⚠️ **Important information about the Timber plugin**
+> With the upcoming release of Timber 2, we will stop releasing Timber as a plugin. We advice everyone to switch to the composer based install as soon as possible.
```

Comment:
I guess we should say something about the fact that v1 will no more supported with PHP 8.2 (has this been decided/announced?).

Reply:
Yes, this is important to note and one of the main reasons why people should make the switch, too.

Reply:
I tried to add it to the notice as well. @jrathert @gchtr. I also clearified that the upgrade guide is only the first step in upgrading to Timber 2.0